### PR TITLE
chore(ci): change scratch image

### PR DESCRIPTION
**Pull request by universal-itengineer**
## Description

Base images are now in files `base-images/base_image_versions.yml` and `base-images/base_virt_image_versions.yml`.
 
For use base images:  
- if werf.yaml use`{{ $.Images.BASE_IMAGE_NAME }}`  
- if image in werf.inc.yaml use `{{ .Images.BASE_IMAGE_NAME }}`  

For example:
```yaml
#werf.inc.yaml
image: {{ $.ImageName }}-original-registry2
from: {{ .Images.BASE_CONTAINER_REGISTRY }}
```
## Why do we need it, and what problem does it solve?

Improve ci.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: images/base-alt-p10/werf.inc.yaml [hunk 1]
```diff
@@ -1,7 +0,0 @@
----
-image: {{ $.ImageName }}
-from: docker.io/alt:p10@sha256:aaddacc787afa546cc03a252adf9799d48f7a0498ecb46ea0d477642b03018eb
-shell:
-  setup:
-  # Create nonroot-user group and user.
-  - groupadd --gid 1001 nonroot-user && useradd nonroot-user --uid 1001 --gid 1001 --shell /bin/bash --create-home
```
Comment (diafour) [inline images/base-alt-p10/werf.inc.yaml:7]: A note from call: use `from: {{.Images.BASE_ALT}}` here and revert werf.inc.yamls for cdi-* and virt-*.

Comment (universal-itengineer) [reply images/base-alt-p10/werf.inc.yaml]: Done

File: werf.yaml [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+
+  {{- range $k, $v := $baseImages }}
+    {{ $baseVirtImagePath := (printf "%s%s" $baseImages.REGISTRY_PATH (trimSuffix "/" $v)) }}
+
+    {{- if ne $k "REGISTRY_PATH" }}
+      {{- $_ := set $.Images $k $baseVirtImagePath }}
+    {{- end }}
+
+  {{- end }}
+---
```
Comment (diafour) [inline werf.yaml:30]:
```suggestion
{{ $_ := set . "Images" dict }}

# Deckhouse Images
{{- $BaseImagesPath := "base-images/deckhouse_image_versions.yml" }}
{{ $baseImages := (.Files.Get $BaseImagesPath | fromYaml) }}

  {{- range $k, $v := $baseImages }}
    {{ $baseImagePath := (printf "%s%s" $baseImages.REGISTRY_PATH (trimSuffix "/" $v)) }}
    {{- if ne $k "REGISTRY_PATH" }}
      {{- $_ := set $.Images $k $baseImagePath }}
    {{- end }}
  {{- end }}

# Base Images
{{- $BaseVirtImagesPath := "base-images/image_versions.yml" }}
{{ $baseImages := (.Files.Get $BaseVirtImagesPath | fromYaml) }}

  {{- range $k, $v := $baseImages }}
    {{ $baseVirtImagePath := (printf "%s%s" $baseImages.REGISTRY_PATH (trimSuffix "/" $v)) }}

    {{- if ne $k "REGISTRY_PATH" }}
      {{- $_ := set $.Images $k $baseVirtImagePath }}
    {{- end }}

  {{- end }}
```

Comment (diafour) [reply werf.yaml:30]: Werf ignores empty docs, but why they needed anyway?

**Review by diafour — APPROVED** [review]

**Pull request merged**